crates/re_string_interner/src/lib.rs

```diff
@@ -166,8 +166,8 @@ impl StringInterner {
 
     pub fn bytes_used(&self) -> usize {
         self.map
-            .iter()
-            .map(|(k, v)| std::mem::size_of_val(k) + std::mem::size_of_val(v) + v.len())
+            .values()
+            .map(|string| std::mem::size_of::<u64>() + std::mem::size_of::<&str>() + string.len())
```

Comment:
size_of_val is nice though since this wouldn't break in a subtle way when the type changes. I'd argue this code got worse

Reply:
…or maybe it wasn't broken, because `size_of_val` actually expects a reference to the data it computes the size of 🤦